csun-metalab/nomi
# NAM-129 -- ENG & UI: Spinner and login feedback

# Description
Added error messages and animated spinner to log in page.
***
# Installation
* `npm install --save vue-loading-spinner`
* `npm install --save-dev babel-preset-stage-2`
***
# Testing
* Verify that spinner appears when "Log In" button pressed.  
* Verify that an error message appears when using a bad username.
